router/xlink_transport/dialer.go

```diff
@@ -34,6 +34,30 @@ type dialer struct {
 	acceptor           xlink.Acceptor
 	transportConfig    transport.Configuration
 	metricsRegistry    metrics.Registry
+	adoptedBinding     string
+}
+
+func (self *dialer) GetHealthyBackoffConfig() xlink.BackoffConfig {
+	return self.config.healthyBackoffConfig
+}
+
+func (self *dialer) GetUnhealthyBackoffConfig() xlink.BackoffConfig {
+	return self.config.unhealthyBackoffConfig
+}
```

Comment:
What are healthy vs unhealthy dial-backoffs?

I assume healthy is for dials that are known to be available to some other part of the deployment and unhealthy are when no components can reach them, but that is a guess.

Reply:
healthy is when the target router is connected to the controller, unhealthy is when it's not. If it's not connected to the controller, it's likely down, but it may not be, so it will still try to connect, just using a different (likely more aggressive) set of backoff parameters